sdk/containerregistry/azure-containerregistry/tests/perfstress_tests/list_artifacts.py

```diff
@@ -0,0 +1,35 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+from azure_devtools.perfstress_tests import PerfStressTest
+from azure.containerregistry import ContainerRegistryClient
+from azure.containerregistry.aio import ContainerRegistryClient as AsyncContainerRegistryClient
+
+
+class ListArtifactsTest(PerfStressTest):
+
+    def __init__(self, arguments):
+        super().__init__(arguments)
+
+        # Create clients
+        account_url = self.get_from_env("CONTAINERREGISTRY_ANONREGISTRY_ENDPOINT")
+        audience = "https://management.azure.com"
+        self.anon_client = ContainerRegistryClient(endpoint=account_url, credential=None, audience=audience)
+        self.async_anon_client = AsyncContainerRegistryClient(endpoint=account_url, credential=None, audience=audience)       
+        self.repository = "node"
+
+    async def global_setup(self):
+        await super().global_setup()
+
+    async def close(self):
+        await self.async_anon_client.close()
+        await super().close()
+
+    def run_sync(self):
+        for manifest in self.anon_client.list_manifest_properties(self.repository):
```

Comment:
How many artifacts is this listing?
The test needs to be repeatable, and comparable across languages - so the actual resources that we run against should have fixed parameters (e.g. "lists X manifest properties in Y seconds")

Reply:
If these resources need to be preconfigured, and not run dynamically - please update the the readme to specify exactly what resources are needed in order for this test to be run in a repeatable way.

Reply:
It needs to pull one image to repo before running this test, it's in README.md section _Import docker images into the newly created registry_, but didn't add a description. (Will do that).

Reply:
> How many artifacts is this listing? The test needs to be repeatable, and comparable across languages - so the actual resources that we run against should have fixed parameters (e.g. "lists X manifest properties in Y seconds")

1, we're using the same artifact across languages.